src/transformers/pipelines/token_classification.py

```diff
@@ -179,6 +189,30 @@ def _sanitize_parameters(
             postprocess_params["aggregation_strategy"] = aggregation_strategy
         if ignore_labels is not None:
             postprocess_params["ignore_labels"] = ignore_labels
+        if process_all is True:
+            if self.tokenizer.is_fast:
+                if stride is not None:
+                    if stride < 0:
+                        stride = 0
+                        warnings.warn(
+                            "`stride` cannot be a negative number, defaulted to" f" `stride={stride}` instead."
+                        )
+                else:
+                    stride = 0
+                tokenizer_params = {
+                    "return_overflowing_tokens": True,
+                    "max_length": self.tokenizer.model_max_length,
+                    "padding": True,
```

Comment:
Remove `padding`.

Reply:
If you remove padding here, you'll have an error: `ValueError: Unable to create tensor, you should probably activate truncation and/or padding with 'padding=True' 'truncation=True' to have batched tensors with the same length. Perhaps your features (`input_ids` in this case) have excessive nesting (inputs type `list` where type `int` is expected).` That's why I kept it

Reply:
Ok, we may need `padding=True`. I thought it wasn't but it could. The main thing to look out for is getting huge batch filled only with padding tokens.